### ISSUE TITLE
Question: why don't  use DetailActivity.IntentBuilder as class name ?

### ISSUE THREAD

**Issue by lsjwzh** [emilsjolander/IntentBuilder#14]
Why don't  use DetailActivity.IntentBuilder as class name ?

**Comment by nathanjones** [comment]
I would prefer this pattern as well. It's what I've been using when I was building similar type safe IntentBuilders manually.

Would you accept a pull request for this?

**Comment by lectricas** [comment]
Yeah, its really uncomfortable to navigate between classes with that sort of names.
@nathanjones try it!
